tests/test_regression.py

```diff
@@ -142,9 +142,9 @@ def test_url(self) -> None:
         class A:
             url: str
 
-        os.environ["p_url"] = "https://github.com/zifeo/dataconf"
-        assert dataconf.env("p", A) == A(url="https://github.com/zifeo/dataconf")
-        os.environ.pop("p_url")
+        os.environ["P_URL"] = "https://github.com/zifeo/dataconf"
+        assert dataconf.env("P", A) == A(url="https://github.com/zifeo/dataconf")
```

Comment:
curious to hear if anything is breaking here?

Reply:
In Windows, os module intentionally sets all keys to uppercase, and this is the reason why in Windows test case fails.